ui/webui/webui-desktop

```diff
@@ -61,7 +60,27 @@ case "$1" in
         ;;
 esac
 
-BROWSER="/usr/bin/firefox --kiosk"
+# prepare empty firefox profile dir with theme based on the passed profile id
+FIREFOX_THEME_DIR="/usr/share/anaconda/firefox-theme"
+FIREFOX_PROFILE_PATH="/tmp/anaconda-firefox-profile"
+THEME_ID=$2
+
+# make sure the profile directory exists and is empty
+if [ -d ${FIREFOX_PROFILE_PATH} ]
+then
+    echo "Cleaning up existing Anaconda Firefox profile directory."
```

Comment:
In what scenario would the profile directory on /tmp already exist?

Reply:
If you launch the installer multiple times on Live media - quite a frequent occurrence during debugging & we know from logs attached to bug reports, that users do it quite often in real life, usually when encountering an error and trying again.

In any case, we really should start with empty profile or else I would expect the behavior on followup starts being non-deterministic given the sheer amount of stuff Firefox dumps to the originally empty folder.